### ISSUE TITLE
Review Entries | Delete button does not work 

### ISSUE THREAD

**Issue by stevendyk** [sillsdev/TheCombine#563]
Plan:
If the last sense is about to be deleted prompt the user 'Deleting the last sense will remove the entry. Do you want to continue? Yes No' If the user selects yes then delete the whole word. If no then do not remove the sense.

Issue #312 failed user test. Reference [ 7/31 #4](https://docs.google.com/document/d/1kUIPMw1SHWXeP6q32QNhxCT0P8RHFSSb7IyclOtvUFI/edit#heading=h.3exr16fj85xi)

**Comment by imnasnainaec** [comment]
Is this referring to deleting recordings?

**Comment by stevendyk** [comment]
No, I don't think so. From the 7/31 Testing Notes:

   1. Found no way to delete an entry. Can only delete a sense.

In Review Entries mode, I wanted to delete a bogus entry I had inserted,
but I found only a way to delete the sense from that entry. An entry
without any sense information is undesirable and perhaps problematic for
reimport into FLEx. When the last remaining sense is deleted from an entry,
the entry should be deleted. Maybe there should also be a trash can icon
where the red circle is in the image below (to delete the entire entry):



<'}}}>< Steven Dyk | 469-619-5677 | stevendyk@gmail.com
<http://www.linkedin.com/profile?viewProfile=&key=5278730&locale=en_US&trk=tab_pro>
